locales-currencies-tz.html

```diff
@@ -0,0 +1,189 @@
+      </p>
+    </emu-clause>
+
+    <emu-clause id="sec-canonicalizetimezonename" aoid="CanonicalizeTimeZoneName">
+      <h1>CanonicalizeTimeZoneName</h1>
+
+      <p>
+        The CanonicalizeTimeZoneName abstract operation returns the canonical and case-regularized form of the _timeZone_ argument (which must be a String value that is a valid time zone name as verified by the IsValidTimeZoneName abstract operation). The following steps are taken:
+      </p>
+
+      <emu-alg>
+        1. Let _ianaTimeZone_ be the Zone or Link name of the IANA Time Zone Database such that _timeZone_, converted to  upper case as described in <emu-xref href="#sec-case-sensitivity-and-case-mapping"></emu-xref>, is equal to _ianaTimeZone_, converted to upper case as described in <emu-xref href="#sec-case-sensitivity-and-case-mapping"></emu-xref>.
+        1. If _ianaTimeZone_ is a Link name, let _ianaTimeZone_ be the corresponding Zone name as specified in the `"backward"` file of the IANA Time Zone Database.
+        1. If _ianaTimeZone_ is `"Etc/UTC"` or `"Etc/GMT"`, return `"UTC"`.
+        1. Return _ianaTimeZone_.
+      </emu-alg>
+
+      <p>
+        The Intl.DateTimeFormat constructor allows this time zone name; if the time zone is not specified, the host environment's current time zone is used. Implementations shall support UTC and the host environment's current time zone (if different from UTC) in formatting.
+      </p>
+    </emu-clause>
+
+    <emu-clause id="sec-defaulttimezone" aoid="DefaultTimeZone">
+      <h1>DefaultTimeZone ()</h1>
+
+      <p>
+        The DefaultTimeZone abstract operation returns a String value representing the valid (<emu-xref href="#sec-isvalidtimezonename"></emu-xref>) and canonicalized (<emu-xref href="#sec-canonicalizetimezonename"></emu-xref>) time zone name for the host environment's current time zone.
+      </p>
+    </emu-clause>
+
+  </emu-clause>
+
+  <emu-clause id="sec-measurement-unit-identifiers">
+    <h1>Measurement Unit Identifiers</h1>
+
+    <p>
+      The ECMAScript 2019 Internationalization API Specification identifies measurement units using a unit identifier as defined by Unicode Technical Standard #35, Part 2, Section 6. Their canonical form is a string containing all lowercase letters with at least one hyphen.
+    </p>
+
+    <p>
+      All well-formed unit identifiers are allowed. However, the set of combinations of unit identifier and language tag for which localized names and symbols are available is implementation dependent. Where a localized name or symbol is not available, the unit identifier itself is used for formatting.
```

Comment:
@littledan Is this OK? You can pass garbage in as your unit identifier as long as it conforms to these rules, like "hello-world", and you get output like "100 hello-world".

Is the fallback reasonable?  If a unit identifier is real but the implementation doesn't have data for it, then just use the unit identifier; for example, "100 length-meter".

Reply:
Thanks for calling this out; I missed it on the earlier review. I'm not sure if this is really the behavior we want. It might be better if the constructor actively signals an error if data is not available, or if the missing data is somehow visible through resolvedOptions(). Maybe we should discuss this in the Intl call. I expected IsWellFormedUnitIdentifier to check whether the unit was in the fixed set that CLDR supports.